R/dbi-result.R

```diff
@@ -139,6 +139,24 @@ setMethod("dbBind", "OdbcResult",
       return(invisible(res))
     }
 
+    paramDfs <- sapply(params, is.data.frame)
+    if (any(paramDfs)) {
+      # When binding table-valued-parameters verify that:
+      # * The `batch_rows` parameter is either unset or equal to one.
+      # * All non-tvp parameters are of length 1.
+      if (!is.na(batch_rows) && batch_rows > 1) {
+        cli::cli_warn("When passing data.frame(s) as query parameters,
+                      the `batch_rows` parameter must be set to 1")
```

Comment:
```suggestion
        cli::cli_warn(c("Since some parameters are data frames, {.arg batch_rows} will be overwritten to `1`.", 
                       "Set {.code batch_rows = 1} to quiet this warning."))
```

Since the code will still run despite the warning, clarify the action taken internally and how to prevent it